web/routes.go

```diff
@@ -63,3 +70,17 @@ func initiateTranscriptionJobHandler(w http.ResponseWriter, r *http.Request) {
 func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }
+
+var statuses = [...]string{
```

Comment:
oop sorry - I changed the way I did it so we shouldn't need this anymore.. I'll delete